### PR TITLE
Pass arguments to subcommand when using call_comand and tenant_command

### LINKED ISSUE #989: Error occurs when passing arguments to subcommand if call_command + tenant_command is used
After #891 was resolved, it is now possible to call `tenant_command` using `call_command`

But it is not possible to pass arguments to subcommand.

Consider the following example:
```python
out = io.StringIO()
call_command(
    'tenant_command',
    'dumpdata',
    'some_app.some_model',
    '--indent=4',
    schema=self.client.tenant.schema_name,
    stdout=out,
)
```
This example fails with `django.core.management.base.CommandError: Error: unrecognized arguments: --indent=4`

Even if we remove `'--indent=4'` from previous example, the result is still not what was expected.\
Although there are no errors this time, `out.getvalue()` produces empty string, not dump data.

### PR DESCRIPTION
**Pull request by hexponent**
Fixes #989

### CODE REVIEW

**Comment by hexponent** [comment]
@tomturner What do you think about this?

**Comment by hexponent** [comment]
@tomturner Please review

**Review by atodorov — CHANGES_REQUESTED** [review]
Can you add a test case exercising the new functionality?

**Comment by hexponent** [comment]
@atodorov Added test case

**Review by atodorov — APPROVED** [review]
One small comment, otherwise LGTM

File: django_tenants/tests/test_commands.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            '{',
+            '    "model": "dts_test_app.dummymodel",',
+            '    "pk": 2,',
+            '    "fields": {',
+            '        "name": "awesome!"',
+            '    }',
+            '}',
+            ']',
+            '',
+        ])
```
[the remark this answers is not in the document]
Comment (hexponent) [reply django_tenants/tests/test_commands.py:33]: Yeah, I agree, I just wanted to omit using another `indent` option in test
Will fix it right away

**Review by atodorov — APPROVED** [review]

**Comment by atodorov** [comment]
@tomturner any objections ?

**Pull request merged**